### DISCUSSION TITLE
`node ace migration:status` return exist code 0 even when migrations are pending

### DISCUSSION THREAD

**Discussion by NathaelB** [adonisjs/core#4827] [Ideas]
When running the `node ace migration:status` command, the exit code is always `0`, even if there are pending migrations. This behavior makes it difficult to use the command in contexts like Kubernetes startup probes, where it is important to know if all migrations have been completed.

I've now written a script that allows me to check the output of the command:

```bash
#!/bin/bash
# Run the migration status command and capture the output
output=$(node ace migration:status)

# Check if the output contains the word "pending"
if echo "$output" | grep -q "pending"; then
  echo "Some migrations are pending:"
  # Extract the names of the pending migrations
  echo "$output" | grep "pending" | awk -F'│' '{print $2}' | sed 's/ //g'
  exit 1
else
  echo "All migrations are completed."
  exit 0
fi
```

Is it possible to return a status error code? I don't know if this could have an impact on other behaviours, or if we could add a flag that would make it easy to find out if the migrations are ready to balance our application's traffic.

**Comment by affaan-m** [comment]
Update: the upgrade issue should now be resolved. The root cause was `plugin.json` and `marketplace.json` being stuck at v1.8.0 — just pushed the fix bumping both to v1.9.0.

To upgrade:
```bash
npx claudepluginhub affaan-m/everything-claude-code --plugin everything-claude-code
```

Or pull latest and run `./install.sh`.

To reiterate: **ECC (the open-source plugin) is and always will be free.** ECC Tools Pro is a completely separate, optional GitHub App for repo analysis. You do not need it to use ECC.

@borzov let me know if the upgrade works for you now.

**Reply by borzov** [reply]
Great, thanks — the issue seems to be resolved now. I’ve updated successfully.
